### ISSUE TITLE
Issue With Spawn Vehicles

### ISSUE THREAD

**Issue by gerung** [ElPumpo/esx#13]
i have issue, vehicle overturns when spawn return vehicles (ladvanced garage and other garage script), but only a big vehicles like truck, bus.

**Comment by ElPumpo** [comment]
You are correct. The native used to spawn vehicles - `CREATE_AUTOMOBILE` can only spawn cars, and should not be used to spawn any other vehicle type.

But I recall it also working for motorcycles and helicopters. The fix to this is to determine the vehicle type and then use different code depending on that.
